faq/faq-overview.md

```diff
@@ -0,0 +1,21 @@
+---
+title: TiDB FAQ Summary
+summary: Summarizes frequently asked questions (FAQs) about TiDB.
+---
+
+# TiDB FAQ Summary
+
+This document summarizes frequently asked questions (FAQs) about TiDB.
+
+|   Category     |           Releted documents          |
+|   :-------   |   :-------------------   |
+|   TiDB architecture and its working principles    |       [TiDB architecture FAQs](/faq/tidb-faq.md)   |
+|   Deployment           |     <ul><li>[Deployment FAQs](/faq/deploy-and-maintain-faq.md)</li><li>[TiUP FAQs](/tiup/tiup-faq.md)</li></ul> |
```

Comment:
How about adding a [TiDB in Kubernetes FAQs](https://docs.pingcap.com/tidb-in-kubernetes/dev/faq) here?

Reply:
Added. Chinese added in https://github.com/pingcap/docs-cn/pull/11100.